ledger-core/v2/vanilla/throw/chain_test.go

```diff
@@ -55,7 +55,7 @@ type errBuilder struct {
 
 func (v errBuilder) _err0() error {
 	err := WithDetails(v.bottomErr, Unsupported())
-	//fmt.Println("0>>>> ", err)
+	// fmt.Println("0>>>> ", err)
```

Comment:
commented debug code x 4

Reply:
it is test, so I see no harm by stashing some extra code. But removed.